index.html

```diff
@@ -498,11 +507,30 @@ <h3>
                 presentation displays connected via the same local area network.
               </p>
               <p class="draft-status">
-                <b>First Public Working Draft</b> <a href=
-                "https://www.w3.org/TR/2021/WD-openscreenprotocol-20210318/">Working Draft</a>
+                <b>Draft state:</b> <a href=
+                "https://www.w3.org/TR/2023/WD-openscreenprotocol-20231016/">Working Draft</a>
               </p>
               <p class="milestone">
-                <b>Expected completion:</b> Q4 2023
+                <b>Expected completion:</b> Q4 2025
+              </p>
+              <p>
+                <b>Reference Draft:</b> <a href=
+                "https://www.w3.org/TR/2021/WD-openscreenprotocol-20210318/">https://www.w3.org/TR/2021/WD-openscreenprotocol-20210318/</a><br>
```

Comment:
Current draft: https://www.w3.org/TR/2023/WD-openscreenprotocol-20231016/

Reply:
Yes, same comment as above, the link targets the First Public Working Draft on purpose.